src/alpha-lab/blink_detection.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Detecting eye blinks using Pupil Lab’s blink detection pipeline"
+description: Perform posthoc and real-time blink detection
+permalink: /alpha-lab/blink-detection/
+tags: [Pupil Invisible, Neon, Realtime Python API]
+---
+
+# Detecting eye blinks using Pupil Lab's blink detection pipeline
+
+<font size = 2>Products used: [Neon](https://pupil-labs.com/products/neon/), [Pupil Invisible](https://pupil-labs.com/products/invisible/), [Realtime Python API](https://pupil-labs-realtime-api.readthedocs.io/en/stable/)</font>
+
+<div class="pb-4" style="display:flex;justify-content:left;">
+  <v-img
+    class="rounded"
+    :src="require('../media/alpha-lab/eye_blinks_anim.gif')"
+    max-width=800px
+  >
+  </v-img>
+</div>
+
+# <font size=4>Eye blinks: artifacts and information</font>
```

Comment:
You can use hash characters ( # ) preceding heading text, like in markdown.
Heading level 1 #
Heading level 2 ##
Heading level 3 ###

So this would be:
## Eye blinks: artifacts and information

Check out the other docs pages for reference